src/account/ordering.js

```diff
@@ -29,13 +29,20 @@ export const sortAccountsComparatorFromOrder = (
   calculateCountervalue: (
     currency: TokenCurrency | CryptoCurrency,
     value: BigNumber
-  ) => ?BigNumber
+  ) => ?BigNumber,
+  starredAccountIds: string[]
```

Comment:
can you make it so it's backward compatible? third param to fallbacks to `[]`

Reply:
Makes sense, yeah I can do that

Reply:
Done, with test 🚀